config/routes.rb

```diff
@@ -36,6 +36,7 @@
   get '/our_services', controller: 'home', action: 'services'
   get '/resources', controller: 'home', action: 'resources'
   get '/training', controller: 'home', action: 'training'
+  get '/project_requirements', controller: 'home', action: 'project_requirements'
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.